src/clustering/extreme_vals.jl

```diff
@@ -20,7 +20,7 @@ function simple_extr_val_sel(data::ClustData,
   elseif rep_mod_method=="append"
     data_mod = input_data_modification(data,idcs)
   else
-    @error("rep_mod_method - "*rep_mod_method*" - does not exist")
+    error("rep_mod_method - "*rep_mod_method*" - does not exist")
```

Comment:
What is the advantage?

Reply:
`@error` is for logging purposes (debugging I guess) and does not throw an exception, whereas `error(msg)` throws an exception. try catch statements work with exceptions. If you want to specify the error type, need to use throw(). For us, I think ErrorException as thrown by error() as the type is ok.